### ISSUE TITLE
Text-only Visits Overview widget

### ISSUE THREAD

**Issue by gaumondp** [piwik/piwik#9433]
Since widgets seems to use only one MySQL CPU per widget, it would reduce the crunching to have text-only (no useless Sparkling) information so I propose to add a new simplier, faster-to-render "Visits Overview".

**Comment by tsteur** [comment]
You mean "Visits overview" widget without sparklines right?

**Comment by gaumondp** [comment]
Yes ! "Visits overview" widget without sparklines. Those double the rendering time and are often "not worth it". ;)
